k8s/namespaces/ia/ia-task-configuration/ithc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-task-configuration
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.java: glob:pr-85-*
```

Comment:
Do we have to tag a PR here?  Could we have not just done something like
`fluxcd.io/automated: "true"`

If it is the same effect then it's fine to keep it as it is.

Reply:
by tagging the PR we can use this PR  to deploy any feature we might need to deploy manually.  In any case, I am not sure it would work without it. This is how it's documented and platops approved it too.

Reply:
Makes sense actually so thanks, good to know!